Warning about audience edit for anon users (#4883)
segmentio/segment-docs: src/engage/audiences/index.md

> warning ""
> It is not possible to edit an audience to convert it from real-time to batch, or vice-versa. If the computation type needs to be changed, you will need to recreate the audience with the appropriate conditions.

> warning ""

> > It is not possible to edit an audience to include anonymous users. If anonymous profiles need to be included, you will need to recreate the audience with the appropriate conditions.

Reviewer: ```suggestion
> You can't edit an audience to include anonymous users. If you need to include anonymous profiles, recreate the audience with the appropriate conditions
```
